docker/core-static/status.json

```diff
@@ -0,0 +1,3 @@
+{
+    "success": true
```

Comment:
is this also present in the other modules?

Reply:
Yes